tests/imodels-access-backend-tests/src/BackendiModelsAccess.test.ts

```diff
@@ -216,6 +218,71 @@ describe("BackendIModelsAccess", () => {
       expect(fs.statSync(localCheckpointFilePath).size).to.be.greaterThan(0);
     });
 
+    it("should query preceding checkpoint v2", async () => {
+      // Arrange
+      const firstNamedVersion = testIModelForRead.namedVersions[0];
+      assert(testIModelFileProvider.changesets.length >= firstNamedVersion.changesetIndex + 1, "Not enough changesets");
+      const nextChangeset = testIModelFileProvider.changesets[firstNamedVersion.changesetIndex];
+      assert(firstNamedVersion.changesetId !== nextChangeset.id, "Unexpected changeset ids");
+     
+      const queryV2CheckpointParams: CheckpointProps = {
+          accessToken,
+          iTwinId,
+          iModelId: testIModelForRead.id,
+          changeset: {
+            id: nextChangeset.id
+          }
+      };
+
+      // Act
```

Comment:
nitpick: I think two `act/assert` blocks would group the logic a bit better:
```
// Act
const v2checkpointForExactChangeset: V2CheckpointAccessProps | undefined = await backendIModelsAccess.queryV2Checkpoint(queryV2CheckpointParams);
// Assert
expect(v2checkpointForExactChangeset).to.be.undefined;

// Act
const v2checkpointForChangesetAllowPrecedingParams = {...queryV2CheckpointParams, allowPreceding: true};
const v2checkpointForChangesetAllowPreceding: V2CheckpointAccessProps | undefined = await backendIModelsAccess.queryV2Checkpoint(queryV2CheckpointParamsAllowPreceding);
// Assert
expect(v2checkpointForChangesetAllowPreceding).to.not.be.undefined;
```

Reply:
done